packages/api/src/router/group.ts

```diff
@@ -236,6 +237,19 @@ const checkSimilarNameAndThrowAsync = async (db: Database, name: string, ignoreI
   }
 };
 
+const throwIfGroupNameIsReservedAsync = async (db: Database, id: string) => {
+  const group = await db.query.groups.findFirst({
```

Comment:
Can't you filter server side here using an ANY? As an example, return the count instead of the group to reduce bandwidth